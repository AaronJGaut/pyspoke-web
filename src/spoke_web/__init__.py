from . import conn
from .version import __version__

