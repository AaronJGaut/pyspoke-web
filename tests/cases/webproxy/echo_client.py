import asyncio
import spoke
import spoke_web

cert = "../../common/localhost.pem"

async def echo(msg):
    print("{}: {}".format(msg.channel, msg.body))

async def main():
    client = spoke.pubsub.client.Client(
        conn_client_class=spoke_web.conn.Client, conn_opts={"cert": cert}
    )
    await client.run()
    await client.subscribe("**", echo)
    await spoke.wait()

asyncio.run(main())

