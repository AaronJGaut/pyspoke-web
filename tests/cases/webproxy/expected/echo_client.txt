foo: 5
foo: 10
